sale_line_partner_description/models/product_buyerinfo.py

```diff
@@ -13,7 +13,6 @@ class ProductBuyerInfo(models.Model):
 
     partner_id = fields.Many2one(
         comodel_name="res.partner",
-        domain=[("customer", "=", True)],
```

Comment:
you could use customer_rank, but ok, the res_partner_search_mode that you put also does the job :)

Reply:
Indeed, I already thought about that option and it would also work but "customer_rank" depends on the "account" module and although our module also depends on "account" indirectly through "sale", I preferred to keep coupling to the minimum possible :-)